HOWTO.md

```diff
@@ -322,6 +322,10 @@ curl -H "token: ${TOKEN}" \
 
 Requesting a parent subproject returns only the objects that live in the parent.
 
+## Using Smart Models
+
+Smart models consist of several models that enable the creation of intricate custom models. To achieve this, it is essential to have smart assets metadata groups in your library. For instance, when working with turbines and foundations, you must initially assign the foundation metadata group to the turbine. This will allow you to select the turbine foundation by modifying the metadata in the designer. Once the objects are linked, all connections to foundation are easily accessible through the turbine.
+
```

Comment:
I don't really get it. On the Wind POC account on `qa` there is no special metadata group, only normal metadata definitions some of which are for smart assets and some of which are not. The assets list on that account is just a list of assets.

So I guess I can deduce that smart assets are just regular assets and we don't need to worry about that part much. All the rest must happen via the metadata.

So let's take an example. How do I get the possible list of foundations for a 5 MW turbine? As far as I can see it is this procedure:

* Get the asset ID for a 5 MW WT asset
* Look up the metadata links and group links for that asset ID
* For any group links, fetch the group definition(s)
* Get all the metadata definitions for the account
* Filter the metadata definitions for which are linked to the 5 MW WT either with a direct link or through a group
* Filter the remaining metadata definitions again for which of them are `type == asset` and `options?.filter?.assetCategories[0] == WindTurbine`
* Then use the results from that to go back to the assets list and filter the assets list by `asset.category == WindTurbine` and `asset.subCategory == metadataDefinition.options?.filter?.assetSubCategories[0]`

and that gives us the 3 possible foundations?